### PR TITLE
docs: add documentation home page

### PR DESCRIPTION
**Pull request by EscapedGibbon**

### CODE REVIEW

**Review by stropitek — COMMENTED** [review]
- You can remove the part about client side. It runs in the browser and in node.JS, but I don't think that needs to be specified since it's implied by being a JS library.
- The lines take too much width (hard to read) and the paragraph is too long. Try to be concise and straight to the point, and keep only what seems essential. If needed, split into 2 paragraphs but that probably already means it's still too long.
- Add all main doc links in the footer (see screenshot)
- The tutorial link in the footer bring to the wrong page.
- Add API link in the footer as well.

CleanShot 2024-06-26 at 08 17 21

**Review by stropitek — COMMENTED** [review]
The page should not have a scrollbar except if you make the height of the window really small.

The second paragraph can be removed.

The text should not exceed 80 characters per line. The text can have a bigger font size since there is nothing else in the page.

**Comment by EscapedGibbon** [comment]
> The page should not have a scrollbar except if you make the height of the window really small.

So I need to fit footer into a page so that there is no scrollbar, correct?

File: docusaurus.config.js [hunk 1]
```diff
@@ -136,39 +136,50 @@ const config = {
...
+                label: 'Glossary',
+                to: '/docs/Glossary',
               },
             ],
           },
           {
             title: 'More',
             items: [
               {
-                label: 'GitHub',
```
Comment (stropitek) [inline docusaurus.config.js:165]: Can you add this link back (to the image-js-typescript repo)?

**Comment by EscapedGibbon** [comment]
Done

File: docusaurus.config.js [hunk 2]
```diff
@@ -170,8 +170,8 @@ const config = {
             title: 'More',
             items: [
               {
-                label: 'API',
-                href: 'https://image-js.github.io/image-js-typescript/',
+                label: 'Github',
```
Comment (stropitek) [inline docusaurus.config.js:173]:
```suggestion
                label: 'GitHub',
```

**Review by stropitek — COMMENTED** [review]
Last thing missing from our discussion: link to API in the footer as well.

**Comment by EscapedGibbon** [comment]
> Last thing missing from our discussion: link to API in the footer as well.

My bad, thought the Github link should replace API link.

**Review by stropitek — APPROVED** [review]
LGTM

I would put the API reference link in the header next to the "Docs" link, instead of next to the dark / light and search box. If possible, move it.

**Pull request merged**